### PR TITLE
[UWP] Enable mouse scrolling for CollectionView 

### LINKED ISSUE #9631: [UWP] How to scroll in CollectionView ?
Hello, I would like to know if it is possible to scroll in a collectionView on UWP without using the ScrollBar? That is to say with a Click and slide or with the mouse wheel?

### PR DESCRIPTION
**Pull request by EmilAlipiev**
### Description of Change ###

Enables mouse scrolling for CollectionView on UWP

### Issues Resolved ### 

- fixes #9631
  
Changed:
 - StructuredItemsViewRenderer.cs 
 
 None

### Platforms Affected ### 
UWP
 
### Testing Procedure ###
Issue9631.cs test case

### CODE REVIEW

**Comment by EmilAlipiev** [comment]
@hartez @samhouts  can we get that fix into next release if possible please? it is very important for uwp. it make no use of collectionview on horizontal if mouse is not working :(

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9631.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+				var collection = new ObservableCollection<Model9631>();
+				var pageSize = 50;
+
+				for (var i = 0; i < pageSize; i++)
+				{
+					collection.Add(new Model9631
+					{
+						Text = "Image" + i,
+						Source = i % 2 == 0
+							? "https://upload.wikimedia.org/wikipedia/commons/thumb/5/5d/Kamchatka_Brown_Bear_near_Dvuhyurtochnoe_on_2015-07-23.jpg/320px-Kamchatka_Brown_Bear_near_Dvuhyurtochnoe_on_2015-07-23.jpg"
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9631.cs:83]: Let's not depend on resources outside of the repository, please!

**Comment by samhouts** [comment]
@EmilAlipiev You can rebase this to 4.8.0. Thanks!

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9631.cs (hunk 1 above)
Comment (EmilAlipiev) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9631.cs:83]: this is not from me. I have copied one of the test case and reused it. it exist in several issue test cases.
https://github.com/xamarin/Xamarin.Forms/search?q=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2Fthumb&unscoped_q=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2Fthumb

Comment (samhouts) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9631.cs:83]: Fair enough :)

Comment (EmilAlipiev) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9631.cs]: Despite that i have changed it now using local icon.

**Comment by EmilAlipiev** [comment]
> @EmilAlipiev You can rebase this to 4.8.0. Thanks!

So this fix should be included in 4.8 pre release or will there be another 4.7 stable version?

**Comment by EmilAlipiev** [comment]
@samhouts is that what you mean? https://github.com/xamarin/Xamarin.Forms/pull/11462

**Comment by samhouts** [comment]
@EmilAlipiev Yes! It looks like there are still some extra commits on that one, though. It might be easier to pull a fresh fork of 4.8.0 and just apply the necessary changes.

**Comment by EmilAlipiev** [comment]
@samhouts  @hartez  @rmarinho  is there anything else i should do here? why cant we get this merged?

**Comment by samhouts** [comment]
@EmilAlipiev There are still too many commits in this PR. Can you try starting over? Sorry about that!

**Comment by EmilAlipiev** [comment]
@samhouts I am closing this one and I created a new one with only 3 files changes 1 commit. There is gif also from the test. please verify this time :) https://github.com/xamarin/Xamarin.Forms/pull/11629

**Pull request closed without merging**